kratos/geometries/geometry.h

```diff
@@ -1386,7 +1386,10 @@ class Geometry
         array_1d<double, 3> normal = Normal(rPointLocalCoordinates);
         const double norm_normal = norm_2(normal);
         if (norm_normal > std::numeric_limits<double>::epsilon()) normal /= norm_normal;
-        else KRATOS_ERROR << "ERROR: The normal norm is zero or almost zero. Norm. normal: " << norm_normal << std::endl;
+        else {
+            std::cout << rPointLocalCoordinates << std::endl;
```

Comment:
I think you pushed this change by error. You can `git checkout` the `master` version of the file.